doc/ensemble.rst

```diff
@@ -0,0 +1,739 @@
+********
+Ensemble
+********
+
+========
+Overview
+========
+An ``Ensemble`` is a SmartSim object comprised of multiple
+:ref:`Model<model_api>` objects, each representing an individual
+simulation. An Ensemble can be be reference as a single instance and
```

Comment:
- reference*d* as
- Maybe "...to launch a group of simulations _simultaneously_" is necessary since they're not independently triggerable?